bin/varnishadm/varnishadm.c

```diff
@@ -467,6 +467,7 @@ main(int argc, char * const *argv)
 		printf(OPTARG "\n");
 		exit(0);
 	}
+	n_arg = getenv("VARNISH_WORKING_DIR");
```

Comment:
Look like something we would want in libvarnishapi instead.

Mandatory bikeshedding: `VARNISH_WORKDIR`

Reply:
> Look like something we would want in libvarnishapi instead.

Yes, just `varnishadm` does not use `VUT`.

> Mandatory bikeshedding: `VARNISH_WORKDIR`

please take a look:

```
$ git grep -Ei 'work.?dir' 
$ git grep -Ei 'working.?dir' 
```

If I did not overlook anything, there is one error message where it is "workdir", otherwise "workdir" is only used in code and all other messages and documentation are for "working directory". I abbreviated to "WORKING_DIR", but probably it should be `VARNISH_WORKING_DIRECTORY`?

Reply:
> Yes, just `varnishadm` does not use `VUT`.

I was referring to `VSM_Attach()`.

> all other messages and documentation are for "working directory". I abbreviated to "WORKING_DIR", but probably it should be `VARNISH_WORKING_DIRECTORY`?

If anything, I'd rather keep environment variables short and have the documentation use the whole "working directory" phrase.